ivy.el

```diff
@@ -1246,6 +1247,12 @@ On error (read-only), call `ivy-on-del-error-function'."
   (unless (= (point) (line-end-position))
     (delete-char arg)))
 
+(defun ivy-delete-forward-char (arg)
+  "Forward to `delete-forward-char' ARG."
+  (interactive "p")
+  (unless (= (point) (line-end-position))
```

Comment:
Why not `eolp`?

Reply:
The other forwarding functions are using `line-end-position` as well; I didn't want to break convention.